usrsctplib/netinet/sctp_pcb.c

```diff
@@ -4173,19 +4171,6 @@ sctp_inpcb_free(struct sctp_inpcb *inp, int immediate, int from)
 	SCTP_INP_WUNLOCK(inp);
 	SCTP_ASOC_CREATE_UNLOCK(inp);
 	SCTP_INP_INFO_WUNLOCK();
-	/* Now we release all locks. Since this INP
```

Comment:
No longer necessary since at this point we are assured no timer referencing us is running.